### PR TITLE
Finished all sections of portfolio and added comments

### PR DESCRIPTION
**Pull request by PriyaVarra**
This is the complete minimum viable product that I planned this week. 

It includes links to my GitHub, LinkedIn, and resume; it contains an about me section with a small description and a photo; it contains a projects section with a small description of each project and links to their corresponding GitHub repos; it contains a hobbies section with photos that can be clicked on to reveal small descriptions; and it contains a gallery section where buttons can be used to cycle through photos with captions.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,12 +13,13 @@
...
-
-
 /**
- * Adds a random greeting to the page.
+ * Changes image and updates caption in gallery section
+ * dir = 1 goes to next image
+ * dir = -1 goes to previous image
  */
 function changePhoto(dir) {
+
```
Comment (wjiamin) [inline portfolio/src/main/webapp/script.js:22]: nit: remove extra line here

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,5 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+/* https://www.w3schools.com/w3css/tryit.asp?filename=tryw3css_templates_cv&stacked=h */
```
Comment (wjiamin) [inline portfolio/src/main/webapp/style.css:11]: Could you move this to the top and add a more detailed comment here?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (PriyaVarra) [reply portfolio/src/main/webapp/script.js]: I have fixed this issue in the commit "Remove space after changePhoto function"

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (PriyaVarra) [reply portfolio/src/main/webapp/style.css]: I have addressed this issue in the "Move reference to top and add more comments"

**Pull request merged**